From: Nicolas Boulenguez <nicolas@debian.org>
Date: Tue, 29 May 2018 21:14:18 +0200
Subject: Disable socket tests

With 0.3.4-1, one test freezes on amd64 pbuilder:
tests/socket_tests-ip.Send_Multicast_V6.

All tests in the socket_tests-ip and socket_tests-unix suites fail on a
mips porterbox.
---
 tests/common_tests.adb | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/tests/common_tests.adb b/tests/common_tests.adb
index 73715c4..11526d2 100644
--- a/tests/common_tests.adb
+++ b/tests/common_tests.adb
@@ -27,8 +27,6 @@ with Ahven.Text_Runner;
 with Type_Tests;
 with OS_Tests;
 with Util_Tests;
-with Socket_Tests.IP;
-with Socket_Tests.Unix;
 with ARP_Tests;
 with UDP_Tests;
 with IP_Tests;
@@ -52,10 +50,6 @@ package body Common_Tests is
                 T     => new OS_Tests.Testcase);
       Add_Test (Suite => Suite.all,
                 T     => new Util_Tests.Testcase);
-      Add_Test (Suite => Suite.all,
-                T     => new Socket_Tests.IP.Testcase);
-      Add_Test (Suite => Suite.all,
-                T     => new Socket_Tests.Unix.Testcase);
       Add_Test (Suite => Suite.all,
                 T     => new ARP_Tests.Testcase);
       Add_Test (Suite => Suite.all,
